apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestUtils.java

```diff
@@ -373,7 +384,9 @@ private static DataSource initUnpooledDataSourceForPostgresql(
   private static DataSource initUnpooledDataSourceForTestContainerWithPostgres(
       String username, String password) {
 
-    String testContainerDatabaseImage = System.getProperty("its.testcontainer.db.image", "");
+    String testContainerDatabaseImage =
+        System.getProperty("its.testcontainer.db.image", TEST_CONTAINER_DATABASE_IMAGE_DEFAULT);
+    LOGGER.debug("Starting container, using image {}", testContainerDatabaseImage);
```

Comment:
These comments were helpful while debugging, so leaving these in as debug-level.